tool/tctl/common/access_request_command.go

```diff
@@ -432,3 +432,10 @@ func printJSON(in interface{}, desc string) error {
 	fmt.Printf("%s\n", out)
 	return nil
 }
+
+func quoteOrDefault(s string, d string) string {
```

Comment:
Nit: You can simplify this signature.
```suggestion
func quoteOrDefault(s, d string) string {
```